generator/src/main/java/de/starwit/generator/generator/EntityImports.java

```diff
@@ -89,7 +89,8 @@ private static void addImportsForRelations(Set<String> imports, Relationship rel
         // imports.add("import javax.persistence.FetchType;");
 
         if (RelationshipType.ManyToMany.equals(relation.getRelationshipType()) ||
-                RelationshipType.ManyToOne.equals(relation.getRelationshipType())) {
+                RelationshipType.ManyToOne.equals(relation.getRelationshipType()) ||
```

Comment:
Meckern auf hohem niveau: `relation.getRelationshipType()` könne man in eine kürzere Var auslagern

Reply:
ja - aber die Idee hier war es, dem gleichen Namen wie bei JHipster zu verwenden